pyflakes/test/test_imports.py

```diff
@@ -184,9 +184,8 @@ def bar():
             def baz():
                 def fu():
                     pass
-        ''',
-                    m.RedefinedWhileUnused, m.RedefinedWhileUnused,
-                    m.UnusedImport, m.UnusedImport)
+        ''', *(m.ImportShadowedByImport, m.RedefinedWhileUnused,
```

Comment:
No worries; I can rework it when I create a narrower patch after #54 has been approved.